lib/src/lints/avoid_returning_widgets/models/avoid_returning_widgets_parameters.dart

```diff
@@ -13,16 +13,10 @@ class AvoidReturningWidgetsParameters {
 
   /// Method for creating from json data
   factory AvoidReturningWidgetsParameters.fromJson(Map<String, dynamic> json) {
-    final exclude = <AvoidReturningWidgetsExclude>[];
-
-    final excludeList = json['exclude'] as Iterable? ?? [];
-    for (final item in excludeList) {
-      if (item is Map) {
-        exclude.add(AvoidReturningWidgetsExclude.fromJson(item));
-      }
-    }
     return AvoidReturningWidgetsParameters(
-      exclude: exclude,
+      exclude: ExcludeParameters.fromJson(
+        excludeList: json['exclude'] as Iterable,
```

Comment:
I think we can make this key reusable, so that all other rules use it consistently.